modules/cooler/dump/main.nf

```diff
@@ -20,18 +20,20 @@ process COOLER_DUMP {
 
     input:
     tuple val(meta), path(cool)
+    val resolution
 
     output:
     tuple val(meta), path("*.bedpe"), emit: bedpe
     path "versions.yml"                  , emit: versions
 
     script:
     def prefix   = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+    def suffix   = resolution?"::$resolution":""
```

Comment:
Can you align this with the above line by `?`, `:` and `=` to improve readability